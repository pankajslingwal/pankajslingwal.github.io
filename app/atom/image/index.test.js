import React from 'react';
import renderer from 'react-test-renderer';
import Image from './';
import EK from '../../assets/EK.png';


describe("Image", () => {

    it('renders correctly, no data', () => {
        const tree = renderer.create(<Image />).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('renders correctly, data passed', () => {
        const props = {
            source:EK,
            alt:"Flight Air India (AI0234) logo"
        };

        const tree = renderer.create(<Image {...props} />).toJSON();
        expect(tree).toMatchSnapshot();
    });

});