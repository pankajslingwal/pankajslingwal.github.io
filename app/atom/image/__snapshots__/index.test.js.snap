// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image renders correctly, data passed 1`] = `
<img
  alt="Flight Air India (AI0234) logo"
  src="test-file-stub"
/>
`;

exports[`Image renders correctly, no data 1`] = `<img />`;
